src/containers/Header/HelpButton.js

```diff
@@ -33,7 +33,7 @@ export default function HelpButton() {
                     <CardHeader className="border-bottom">
                         <div className="card-header-title">
                             <i className="cil-info pr-2" />
-                            Help Card
+                            {t("HelpButton|Help Card")}
```

Comment:
@Aringoaway Dont use `Card` in the title.... as I wrote it here: https://github.com/TeskaLabs/asab-webui/pull/393#discussion_r1134115491